cmd/ocm-backplane/login/login.go

```diff
@@ -183,7 +185,12 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	logger.Debugf("Using backplane URL: %s\n", bpURL)
-
+	backplanehost, err := getbackplanehost(bpURL)
+	if err != nil {
+		return err
+	}
+	logger.Debugf("Backplane URL resolves to %s \n", backplanehost)
+	logger.Debugf("Please refer to https://source.redhat.com/groups/public/sre/wiki/backplane_troubleshooting to figure out the associated backplane server")
```

Comment:
```
if err != nil {
		logger.Errorf(err)
	} else{
	logger.Debugf("Backplane URL resolves to %s \n", backplanehost)
	logger.Debugf("Please refer to https://source.redhat.com/groups/public/sre/wiki/backplane_troubleshooting to figure out the associated backplane server")
}
```

Reply:
@supreeth7 Does the suggested change look okay?

Reply:
@devppratik :
c68b68292e2f72491f32804011d0946fa710fac6  contains your suggested change.
The 'make test' passes successfully except for the following known issue reported in the main branch too.